xcode/Subconscious/Shared/Components/Feed/Feed.swift

```diff
@@ -112,28 +112,24 @@ struct FeedView: View {
             app.actions.compactMap(FeedAction.from),
             perform: store.send
         )
-    }
-}
-
-extension FeedAction {
-    static func from(_ action: AppAction) -> Self? {
-        switch action {
-        case .succeedIndexOurSphere:
-            return .refreshAll
-        case .succeedIndexPeer:
-            return .refreshAll
-        case .succeedRecoverOurSphere:
-            return .refreshAll
-        case .requestFeedRoot:
-            return .requestFeedRoot
-        default:
-            return nil
+        /// Replay some feed actions on app store
+        .onReceive(
+            store.actions.compactMap(AppAction.from),
+            perform: app.send
+        )
+        .onReceive(store.actions) { action in
+            FeedAction.logger.debug("\(String(describing: action))")
```

Comment:
Feed was previously not logging its actions.

Note to self: we should get logging into `ObservableStore.Store` as a core feature, to avoid this requirement.